### DISCUSSION TITLE
Cannot post png screenshot of academic registration letter

### DISCUSSION THREAD

**Discussion by EvergreenTor** [community/community#110654] [GitHub Education]
### Hi everyone,

when I try and submit a screenshot of my academic letter at the following page https://education.github.com/discount_requests/13817434/additional_information
when I inspect, I see a HTTP post error for a split second. The submit button gets disabled. This error makes it so my screenshot does not even get processed.

**Comment by EvergreenTor** [comment]
I tried again at https://education.github.com/discount_requests/13930051/additional_information and got the same error. Can someone please help?

**Comment by SebastienXia** [comment]
## Hi, @EvergreenTor 

Did you use the option *download a file* ?
